src/test/java/org/ays/auth/controller/AysAuthEndToEndTest.java

```diff
@@ -161,7 +164,7 @@ void givenValidForgotPasswordRequest_whenUserHasNotPasswordAndPasswordCreatedWit
         // Then
         String endpoint = BASE_PATH.concat("/password/forgot");
         MockHttpServletRequestBuilder mockHttpServletRequestBuilder = AysMockMvcRequestBuilders
-                .post(endpoint, mockForgotPasswordRequest);
+                .post(endpoint, userToken.getAccessToken(), mockForgotPasswordRequest);
```

Comment:
Public endpointlerde token göndermiyoruz
```suggestion
                .post(endpoint, mockForgotPasswordRequest);
```